samples/Calculator/csharp/windows/calculator.sln

```diff
@@ -3,7 +3,10 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29215.179
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "calculator", "calculator\calculator.csproj", "{59D6F7C8-E336-426F-87B9-3EAB99E3631E}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "calculator", "calculator\calculator.vcxproj", "{AE36F0CC-A488-4E84-927F-4E821BA133FC}"
```

Comment:
It looks like you accidentally created another C++ calculator, rather than  C#.

Reply:
Good catch! This came from me just copy/pasting the upgrade steps, have since updated them to include the `--language cs` option :)